Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -4583,7 +4584,7 @@ void AXObjectCache::deferRecomputeIsIgnoredIfNeeded(Element* element)
         m_deferredRecomputeIsIgnoredList.add(*element);
         return;
     }
-    recomputeIsIgnored(element->renderer());
```

Comment:
can we store element->renderer() in a local variable so we don't have to access twice?

Reply:
Fixed!